controllers/feeds/getAnonymousFeedChat.js

```diff
@@ -57,6 +58,7 @@ const getAnonymousFeedChatService = async (req, res) => {
           totalComment: totalComment + commentLevel2 + total3,
           isOwnPost,
           isOwnSignedPost,
+          isOwnAnonymousPost,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `isOwnAnonymousPost` in the returned object is a good change as it provides more context about the post ownership. However, please ensure that this variable is properly initialized and used where necessary.

```diff
+ isOwnAnonymousPost,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->